_events/community-meeting.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Community meeting
+summary: Our monthly meeting to discuss anything AntennaPod. Join us!
+image:
+permalink: /events/:slug
+date: "2022-04-09 18:00:00 +0200" # yyyy-mm-dd hh:mm:ss _hhmm Where _ is either + or - to indicate the timezone in which the event is noted
+ICS: 0361F458-FD97-46CD-A014-076C1995439E # Name of file added in assets/events folder
```

Comment:
Instead of having one ICS file per event that one needs to add to their calendar, how about having one single, dynamic ICS that contains all (old and new) events? We could even automate the creation of that file, based on the metadata given above. Then one needs to add it to their calendar only once, and automatically gets new events showing up